test/Makefile.inc

```diff
@@ -20,6 +20,8 @@ AM_CFLAGS = $(CUNIT_CFLAGS)
 
 AM_LDFLAGS = -L$(LIB) -static
 
+LOG_COMPILER = $(top_srcdir)/platform/@with_platform@/test/wrapper-script.sh
```

Comment:
linux-generic platform does not have this wrapper-script.sh

Reply:
That's why I added an empty file that doesn't effect the test run.